ReviewMonitor/Apps List View Controller/App Detail View Controller/MetaViewController.swift

```diff
@@ -0,0 +1,89 @@
+//
```

Comment:
Can we rename this file to `AppMetadataViewController.swift`? It's more clear.

Reply:
Ok

Reply:
Done